lib/membrane/remote_controlled/pipeline.ex

```diff
@@ -0,0 +1,426 @@
+defmodule Membrane.RemoteControlled.Pipeline do
+  @moduledoc """
+  `Membrane.RemoteControlled.Pipeline` is a basic `Membrane.Pipeline` implementation that can be
+  controlled by a controlling process. The controlling process can request the execution of arbitrary
```

Comment:
The first sentence till two break lines serves as a short description in the modules list in docs
```suggestion
  A basic `Membrane.Pipeline` implementation that can be controlled by a controlling process.

  The controlling process can request the execution of arbitrary
```

Reply:
done